src/django_components/component_media.py

```diff
@@ -683,26 +699,33 @@ def _is_media_filepath(filepath: Any) -> bool:
     return False
 
 
-def _normalize_media_filepath(filepath: ComponentMediaInputPath) -> Union[str, SafeData]:
-    if callable(filepath):
-        filepath = filepath()
+def _normalize_media_filepath(filepaths: Sequence[ComponentMediaInputPath]) -> List[Union[str, SafeData]]:
```

Comment:
Before, we went file path by file path, resolving it relative to the component file if possible.

Some functions like this `_normalize_media_filepath` received only a single path, and returned it mapped.

But with support of globs, a single glob path may be expanded to multiple file paths. So some changes are to the mapping logic. And e.g. this `_normalize_media_filepath` now receives a whole list, and returns a whole list.